services/core/shared/core/compat/sdk_v5/voters.js

```diff
@@ -104,23 +120,13 @@ const getVoters = async params => {
 		params.receivedAddress = getBase32AddressFromHex(extractAddressFromPublicKey(publicKey));
 	}
 
-	const resultSet = await votesDB.find({ sort: 'timestamp:desc', receivedAddress: params.receivedAddress });
-	if (resultSet.length) {
-		params.ids = resultSet.map(row => row.id);
-
-		const response = await getTransactionsByIDs(params.ids);
-		if (response) {
-			const voteMultiArray = response
-				.map(tx => tx.asset.votes.map(v => ({ ...v, senderPublicKey: tx.senderPublicKey })));
-			let allVotes = [];
-			voteMultiArray
-				.forEach(lvotes => allVotes = allVotes.concat(lvotes.map(v => ({
-					...normalizeVote(v),
-					sentAddress: getBase32AddressFromHex(extractAddressFromPublicKey(v.senderPublicKey)),
-				}))));
-			votes.data.votes = allVotes;
-		}
+	let resultSet;
+	if (params.aggregate) {
+		resultSet = await votesAggregateDB.find({ sort: 'timestamp:desc', receivedAddress: params.receivedAddress });
```

Comment:
The logic currently does not support `limit` and `offset`. Please make the necessary changes.

Reply:
It already support `limit` and `offset`.